include/crow/middlewares/cookie_parser.h

```diff
@@ -93,14 +93,14 @@ namespace crow
             // Expires attribute
             Cookie& expires(const std::tm& time)
             {
-                expires_at_ = time;
+                expires_at_ = std::unique_ptr<std::tm>(new std::tm(time));
```

Comment:
Change it to `make_unique`?

Reply:
I believe `std::make_unique` wasn't added till `C++14`.

Reply:
So maybe it is the time to bump up at least to C++14? ;)

Reply:
@mrozigor is `make_unique` different from what we have here?

Reply:
Basically not, but it's nicer.

Reply:
Then I would be against deprecating C++11 support. I know it's old, but there's too much infrastructure still relying on it.

Reply:
Well github just deleted my comment...

C++14 breaks compatibility for minor changes, this is a problem because downstream projects may not be able to just simply upgrade to a new standard and possibly a new compiler too. We've already run into issues with GCC 4.8 and GCC 5 support (I believe we'll end up dropping 4.8 soon because of #463).

The work's already been done, and it seems to be a lot more leak-proof than using simple pointers. That being said if you're still not convinced @mrozigor  we can perhaps use a macro or function that uses this implementation for C++11 and `std::make_unique` for later versions.

Reply:
No, we can merge it as is. I'm just curious if is there anyone who is not able to update to newer gcc/clang? Even our IoT project was able to use GCC in version 9.

Anyway, it's just my thought about this, feel free to ignore it ;)

Reply:
Actually we had someone who wanted to downgrade GCC from 4.8 all the way to 4.4 [on gitter](https://gitter.im/crowfork/community?at=62a4cceedeea5616bbf1a292) 😅. I eventually had to turn him/her down because 4.4 didn't properly support c++11.

Reply:
TBH we shouldn't keep older C++ version just because someone is using very old software ;)